api4/ldap_test.go

```diff
@@ -22,6 +26,8 @@ func TestSyncLdap(t *testing.T) {
 	th := Setup().InitBasic()
 	defer th.TearDown()
 
+	th.App.SetLicense(model.NewTestLicense("ldap_groups"))
```

Comment:
Could we add the test case for `th.SystemAdminClient.SyncLdap()` failing without the license?